jep/226/README.adoc

```diff
@@ -1,4 +1,4 @@
-= JEP-0000: External Fingerprint Storage
+= JEP-226: External Fingerprint Storage
```

Comment:
@oleg-nenashev will you update https://www.jenkins.io/sigs/cloud-native/pluggable-storage/ to match?

Reply:
Thanks for pointing it out @jglick, Ill submit a PR to update it

Reply:
Somebody will do it, if course.

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/3449